demos/basics/loading-networkx.ipynb

```diff
@@ -0,0 +1,984 @@
+{
```

Comment:
<p>not caused by this notebook, but perhaps our info() output should indicate that the edges are weighted?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1074/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Yup, I had the same thought when working on this notebook: https://github.com/stellargraph/stellargraph/issues/1069